web/template/layout/mlpa-case-summary.gohtml

```diff
@@ -0,0 +1,60 @@
+{{ define "mlpa-case-summary" }}
+    <div class="app-!-caseworker-summary">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+                <div class="govuk-!-display-block">
+                    <h3 class="govuk-heading-s govuk-!-display-inline govuk-!-padding-right-3">Case summary</h3>
+                    <button type="button" class="govuk-accordion__show-all govuk-!-margin-0" data-module="app-case-summary-toggle">
+                        <span class="govuk-accordion-nav__chevron govuk-accordion-nav__chevron--up case-summary-toggle-icon"></span>
+                        <span class="case-summary-toggle-text">Hide</span>
+                    </button>
+                </div>
+            </div>
+        </div>
+
+        <div class="govuk-grid-row case-summary">
+            <div class="govuk-grid-column-one-third">
+                <div class="govuk-!-display-block">
+                    <svg class="app-!-svg-icon" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 -2 19 19" height="20px" width="20px" overflow="visible">
```

Comment:
Rather than putting in a specific height:
```suggestion
                    <svg class="app-!-svg-icon" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 -2 19 19" overflow="visible">
```
I think it should just be matched to the height of the text surrounding it:
```css
.app-svg-icon {
  height: 1em;
}
```
That gives more flexibility for different type sizes, and makes sure it works if the user increases/reduces the font size.